lib/koans/15_protocols.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Protocols do
+  use Koans
+
+  defprotocol School do
+    def enrol(name)
```

Comment:
Haha, I figured that was the case. Listen ya'll are lucky to get something legible out of this Alabama boy (see what I did there? :wink:)

@ukutaht and I had a conversation about this before: https://github.com/elixirkoans/elixir-koans/pull/75. I'm not deeply opinionated one way or the other 😺